lib/pages/chat/input_bar.dart

```diff
@@ -307,172 +308,178 @@ class InputBar extends StatelessWidget with PasteImageMixin {
 
   @override
   Widget build(BuildContext context) {
-    final useShortCuts = (PlatformInfos.isWeb ||
-        PlatformInfos.isDesktop ||
-        AppConfig.sendOnEnter);
-    return RawKeyboardListener(
-      focusNode: keyboardFocusNode,
-      onKey: (event) {
-        if (useShortCuts && event.isEnter) {
-          onSubmitted?.call(controller?.text ?? '');
+    return CallbackShortcuts(
+      bindings: {
+        SingleActivator(
+          flutter.LogicalKeyboardKey.keyV,
+          meta: PlatformInfos.isMacKeyboardPlatform,
+          control: !PlatformInfos.isMacKeyboardPlatform,
+        ): () async {
+          if (await Clipboard.instance.isReadableImageFormat()) {
```

Comment:
please refactor
- Move it to function
- Move TypeAHead to private class
- Remove delay when send because it no need anymore

Reply:
Okay i have refactor it, but a diff idea, and we can't remove the delay because of an error double message sent when typing vietnamese. The debouncer is for shift+enter only